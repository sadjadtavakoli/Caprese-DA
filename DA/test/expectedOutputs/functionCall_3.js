
#1 function anonymous11 is called with variables args by functionCall_3.js
#1 function anonymous11 entered with variables args from functionCall_3.js
#1 function anonymous11 exited with return values undefined to function functionCall_3.js
#end Execution