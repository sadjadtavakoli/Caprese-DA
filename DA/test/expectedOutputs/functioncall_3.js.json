{"arrowAnonymousFunction-test/unit_tests/functioncall_3.js-1-5":{"impacted":["t_1"]},"keyMap":{"t_1":"functioncall_3.js-test/unit_tests/functioncall_3.js-1-5","t_2":"arrowAnonymousFunction-test/unit_tests/functioncall_3.js-1-5"}}